packages/twilio/src/conversations/index.ts

```diff
@@ -2,9 +2,10 @@ import TwilioConversation from './TwilioConversation';
 
 export const useTwilioConversation = (
   accountSid: string = process.env.TWILIO_ACCOUNT_SID,
-  authToken: string = process.env.TWILIO_AUTH_TOKEN
+  authToken: string = process.env.TWILIO_AUTH_TOKEN,
+  serviceId: string = process.env.TWILIO_CHAT_SERVICE_SID
 ) => {
-  return new TwilioConversation({ accountSid, authToken });
+  return new TwilioConversation({ accountSid, authToken, serviceId });
```

Comment:
Simplify to `() => (new TwilioConversation({ accountSid, authToken, serviceId }))` ?

Reply:
done